### PR TITLE
Home page rework

### PR DESCRIPTION
**Pull request by aeolianeth**
## What does this PR do and why?

Related to https://github.com/jbx-protocol/juice-interface/issues/1059

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by aeolianeth** [comment]
@johnnyd-eth this is ready for your review 🙏

File: src/components/Landing/TopProjectsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+          <div>
+            <h2
+              style={{
+                fontSize: '2.5rem',
+                textAlign: 'center',
+                fontWeight: 600,
+                marginBottom: '0.8rem',
+              }}
+            >
+              Fund and operate your thing, your way.
```
Comment (johnnyd-eth) [inline src/components/Landing/TopProjectsSection.tsx:112]: Caitlyn Jenner dat shit

File: src/components/Landing/index.tsx [hunk 2]
```diff
@@ -104,9 +68,9 @@ export default function Landing() {
         <Trans>Built for:</Trans>
       </p>
       {[
-        t`DAOs and communities`,
```
Comment (johnnyd-eth) [inline src/components/Landing/index.tsx:107]: **Praise: ** excellent

File: src/components/Landing/TopProjectsSection.tsx [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+              style={{
+                textAlign: 'center',
+                fontSize: '1rem',
+                marginBottom: '0.3rem',
+                color: colors.text.primary,
+              }}
+            >
+              <Trans>
+                Juicebox puts the fun back in funding so you can focus on
+                building.
```
Comment (johnnyd-eth) [inline src/components/Landing/TopProjectsSection.tsx:125]: Love this

**Review by johnnyd-eth — CHANGES_REQUESTED** [review]
Amazing shit, massive improvements. Extremely bullish. Left a few more comments in Discord here: https://discord.com/channels/775859454780244028/989781720817627217/991586837116231781

**Review by johnnyd-eth — APPROVED** [review]
LVGTM.

**Pull request merged**